scripts/ocm/ocm.sh

```diff
@@ -157,7 +157,7 @@ install_addon() {
 
     # Add mandatory "cidr-range" and "addon-managed-api-service" (quota) params with default value in case of rhoam (managed-api-service) addon 
     if [[ "${addon_id}" == "managed-api-service" ]]; then
-    	addon_payload="{\"addon\":{\"id\":\"${addon_id}\"}, \"parameters\": { \"items\": [{\"id\": \"cidr-range\", \"value\": \"10.1.0.0/26\"}, {\"id\": \"addon-managed-api-service\", \"value\": \"${QUOTA}\"}] }}"
+    	addon_payload="{\"addon\":{\"id\":\"${addon_id}\"}, \"parameters\": { \"items\": [{\"id\": \"cidr-range\", \"value\": \"10.1.0.0/26\", \"id\": \"addon-resource-required\", \"value\": true }, {\"id\": \"addon-managed-api-service\", \"value\": \"${QUOTA}\"}] }}"
```

Comment:
I haven't tested it, but should it look like this
```suggestion
    	addon_payload="{\"addon\":{\"id\":\"${addon_id}\"}, \"parameters\": { \"items\": [{\"id\": \"cidr-range\", \"value\": \"10.1.0.0/26\"}, {\"id\": \"addon-resource-required\", \"value\": true }, {\"id\": \"addon-managed-api-service\", \"value\": \"${QUOTA}\"}] }}"
```

Reply:
Actually I just tested it out and it seems that also parameter's value have to be always a type of string
```
{
  "kind": "Error",
  "id": "400",
  "href": "/api/clusters_mgmt/v1/errors/400",
  "code": "CLUSTERS-MGMT-400",
  "reason": "Bad request body: addon parameter(s) 'value' must be of type 'string'",
  "operation_id": "5c937a88-d283-46dd-906b-3d975278a7fa"
}
```
so
```suggestion
    	addon_payload="{\"addon\":{\"id\":\"${addon_id}\"}, \"parameters\": { \"items\": [{\"id\": \"cidr-range\", \"value\": \"10.1.0.0/26\", \"id\": \"addon-resource-required\", \"value\": "true" }, {\"id\": \"addon-managed-api-service\", \"value\": \"${QUOTA}\"}] }}"
```

Reply:
:+1:

Reply:
(just realized I forgot about backslashes 🤦  updated ^)